source/content/guides/support/02-contact-support.md

```diff
@@ -25,58 +25,104 @@ We recommend you begin your Support journey from the affected [Personal Workspac
+### From the workspace
+1. [Go to the workspace](/guides/account-mgmt/workspace-sites-teams/workspaces#switch-between-workspaces).
 
-1. Click **Support**, **Launch Chat**, then **New Conversation**.
+1. Go to the **Support** tab, then select **Start chat**.
 
-1. Provide a detailed summary of your support request, such as:
-   - Steps to reproduce the issue (including URLs or paths to files).
-   - Environments affected (Multidev/Dev/Test/Live), if applicable.
-   - When the issue began.
-   - Error messages received, if applicable.
-   - Links to screenshots or screencasts of the behavior, if necessary.
+1. Provide a [detailed description](#provide-a-detailed-description) of your support request.
 
 You can download a transcript of your chat with the <Icon icon="arrowDownToLine"/> button:
 
 ![Screenshot of a the support chat window with an arrow pointing to the button to download a transcript.](../../../images/chat-log-download.png)
 
 <Alert title="Note" type="info">
 
-Some in-browser ad or tracking blockers can interfere with the **Launch Chat** button. If you don't see it, try allowlisting `pantheon.io` or disabling the browser extension. Intercom has posted more details on [why this happens](https://www.intercom.com/help/en/articles/2546256-troubleshooting-when-the-messenger-doesn-t-appear) on their site.
+Some in-browser ad or tracking blockers may interfere with the **Start chat** button. If you don't see the button, try allowlisting `pantheon.io` or disabling the browser extension. Intercom has provided more details on [why this happens](https://www.intercom.com/help/en/articles/2546256-troubleshooting-when-the-messenger-doesn-t-appear) on their site.
 
 </Alert>
 
-## Premium Technical Support and On-Call Services
+### From the site dashboard
+
+1. [Go to the site dashboard](/guides/account-mgmt/workspace-sites-teams/sites#site-dashboard).
+
+1. Go to the **Support** tab, then select **Launch Chat**.
+
+1. Provide a [detailed description](#provide-a-detailed-description) of your support request.
+
+## Emergency ticket
+**Diamond** and **Platinum** customers also have the option to open an **emergency ticket** (formerly On-call help) for business-critical issues. Pantheon escalates emergency tickets to our support engineers, even during off-hours. If your issue is non-business-critical, please open a [general support](#general-support-ticket) ticket instead.
+
+### From the workspace
+1. [Go to the workspace](/guides/account-mgmt/workspace-sites-teams/workspaces#switch-between-workspaces).
+
+1. Go to the **Support** tab, then select **Open emergency ticket**.
+
+1. Enter a subject (summary of your issue).
+
+1. Provide a [detailed description](#provide-a-detailed-description) of your support request.
+
+1. Click **Submit emergency ticket**
```

Comment:
Right now the above section is not accurate because phoebe still has "on-call" copy that [has not been updated in the submission form ](https://github.com/pantheon-systems/dashboard/pull/3240/files#diff-164e1f655be6b899b57a6e0142896ef02428d5c11fe966eced4d36ae834b250eR3-R10). I think that's an oversight though and needs to be completed as part of this release. I raised to @stephpawlowski for review in docs slack thread